qa/cephfs/objectstore-ec/bluestore-comp-ec-root.yaml

```diff
@@ -0,0 +1,28 @@
+overrides:
+  thrashosds:
+    bdev_inject_crash: 2
+    bdev_inject_crash_probability: .5
+  ceph:
+    fs: xfs
```

Comment:
Isn't this `fs: xfs` (here and elsewhere) redundant for bluestore?

Reply:
Maybe it's for the small (100MB?) XFS partition each BlueStore OSD seems to have?

Reply:
Oh I know.. ti's because teuthology is doing its own special thing ot set up OSDs.  this is the fs we format the device with.  It then manually creates an osd instance there.  For bluestore, the "block" device is just a file called 'block' that is most of the device (size specified in bluestore.yaml).  This line just makes sure the containing device is formatted with xfs and not something else.